packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -48,12 +48,17 @@ describe('AppSec Index', () => {
         eventTracking: {
           enabled: true,
           mode: 'safe'
+        },
+        apiSecurity: {
+          enabled: false,
+          requestSampling: 0
         }
       }
     }
 
     web = {
-      root: sinon.stub()
+      root: sinon.stub(),
+      getContext: sinon.stub()
```

Comment:
what's all those getContext for ? i don't understand where they're used

Reply:
Probably remainings from a previous implementation. Removing it.